ci/python_test.sh

```diff
@@ -4,21 +4,22 @@ set -euxo pipefail
 
 VENV=env
 
-rm -rf "$VENV"
-python3 -mvenv "$VENV"
+if [[ "$OSTYPE" != "msys" ]]; then
```

Comment:
Why only for `msys` now? Before it was for all OS.

Reply:
It's exept for `msys`, not only :)

Reply:
Yes, sorry. I'm useless here... :(

Reply:
I think the reason for msys should be described, so we know what it is, and why its there